Aztec/Classes/TextKit/ParagraphStyle.swift

```diff
@@ -419,4 +429,29 @@ extension ParagraphStyle {
             }
         }
     }
+    
+    // MARK: - Swizzling NSparagraphStyle
+    
+    /// We need to Swizzle NSParagraphStyle's `isEqual` because it does not consider the class for the comparison.
+    /// This is probably because `NSParagraphStyle` was never intended for subclassing, but since we subclassed it
+    /// we need it to recognize the difference.
+    ///
+    private static func swizzleSuperclass() {
+        guard let originalMethod = class_getInstanceMethod(NSParagraphStyle.self, #selector(isEqual(_:))),
+            let swizzledMethod = class_getInstanceMethod(NSParagraphStyle.self, #selector(swizzledIsEqual(_:))) else {
+                return
+        }
+        
+        method_exchangeImplementations(originalMethod, swizzledMethod)
+    }
+}
+
+extension NSParagraphStyle {
+    @objc func swizzledIsEqual(_ object: Any?) -> Bool {
+        guard object_getClass(self) == object_getClass(object) else {
```

Comment:
What happens when you compare a NSParagraphStyle with a NSMutableParagraphStyle?

Reply:
Well worst case scenario they're not considered equal (which I don't think is necessarily a bad thing).

That said, I'll try reviewing this logic for that scenario.